Sources/Utility/Git.swift

```diff
@@ -22,15 +22,15 @@ extension Version {
 }
 
 public class Git {
-    /// Compute the version -> tag mapping from a list of input `tags`.
-    public static func convertTagsToVersionMap(_ tags: [String]) -> [Version: String] {
+    /// Compute the version -> tags mapping from a list of input `tags`.
+    public static func convertTagsToVersionMap(_ tags: [String]) -> [Version: [String]] {
         // First, check if we need to restrict the tag set to version-specific tags.
-        var knownVersions: [Version: String] = [:]
+        var knownVersions: [Version: [String]] = [:]
         for versionSpecificKey in Versioning.currentVersionSpecificKeys {
             for tag in tags where tag.hasSuffix(versionSpecificKey) {
                 let specifier = String(tag.dropLast(versionSpecificKey.count))
                 if let version = Version(string: specifier) ?? Version.vprefix(specifier) {
```

Comment:
As we're always doing `Version(string: specifier) ?? Version.vprefix(specifier)`, couldn't we profit from this PR to remove `vprefix` and create a single failable initializer that checks for both formats? Sounds more Swifty:

```swift
extension Version {
    init(tag: String) -> Version? {
        if string.first == "v" {
            return Version(string: String(string.dropFirst()))
        } else {
            return Version(string: string)
        }
    }
}
```

Reply:
Ah, yes that seems better!